traffic_ops/app/lib/API/Server.pm

```diff
@@ -947,6 +952,10 @@ sub details_v11 {
 		}
 
 		my $rs_ds_data = $row->deliveryservice_servers;
+		#FOR THE REVIEWER - Currently I do not check DS tenancy here.
```

Comment:
always check tenancy in my opinion if use_tenancy=1. never rely on roles / capabilities because roles can be setup to include whatever capabilities the system owner wants. there is no harm in always ensuring the user has the proper tenancy. they can always sidestep tenancy by assigning themself with the root tenant...